nuxeo-webdav-test/src/test/java/org/nuxeo/ecm/webdav/WebDavClientTest.java

```diff
@@ -419,6 +440,85 @@ public void testLockUnlockNoAccept() throws Exception {
         testLockUnlock(null);
     }
 
+    @Test
+    public void testMSOfficeSaveFileFlow() throws Exception {
+        // create a fake bin tmp file which will finally be a docx file
+        String basePath = "/workspaces/workspace/";
+        String rootUri = TEST_URI + basePath;
+        String origName = "Document.docx";
+        String octetStreamMimeType = MimetypeRegistry.DEFAULT_MIMETYPE;
+        byte[] bytes = "Fake BIN".getBytes("UTF-8");
```

Comment:
`import static java.nio.charset.StandardCharsets.UTF_8` and use `getByes(UTF_8)`